sci-libs/libgeodecomp/libgeodecomp-0.4.0-r1.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
```

Comment:
Since you are already working on it, could you please also bump this ebuild to `EAPI-7` and `cmake.eclass`? :)
Should be mostly straightforward with the `doc` block moving into `BDEPEND` and running `s/cmake-utils/cmake/` (except for `cmake-utils_src_make` needing to be renamed to `cmake_build`).